src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    return (days * 40) - 20;
+  }
+
+  if (days >= 7) {
+    return (days * 40) - 50;
+  }
```

Comment:
```suggestion
  return (days * 40) - 50;
```
Этот if не нужен, если предыдущие два не сработали - значит days точно больше или равно 7